grommet/grommet
# migrate formfield tests from js to tsx

What does this PR do?
This PR migrates FormField component tests from *.js to *.tsx to allow ts-jest to catch type-related errors in the unit testing pipeline. Only one change made to support this.

Where should the reviewer start?
Ensure all tests are currently passing

What testing has been done on this PR?
Manually ran and passed FormField component tests locally.
CircleCI tests have passed.

How should this be manually tested?
Modified tests behave identically as before.

Do Jest tests follow these best practices?

Any background context you want to provide?

What are the relevant issues?
Fixes #5599 for FormFields

Screenshots (if appropriate)

Do the grommet docs need to be updated?
no

Should this PR be mentioned in the release notes?
no

Is this change backwards compatible or is it a breaking change?
compatible
